torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -90,15 +90,21 @@ def vertical_flip(inpt: features.InputTypeJIT) -> features.InputTypeJIT:
 vflip = vertical_flip
 
 
+def _compute_resized_output_size(
+    image_size: Tuple[int, int], size: List[int], max_size: Optional[int] = None
+) -> List[int]:
+    if isinstance(size, int):
+        size = [size]
```

Comment:
I took the liberty to put this into the common function that is called from every kernel.